Sources/CombineWamp/PeerWorkflow/WampSerializing.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+public struct WampSerializing {
```

Comment:
Nitpick: `ing` suffixes are usually used for protocols, IIRC.

Reply:
Usually I follow this guideline:
- protocol: *able
- protocol witness (de-protocolized contract): *int

Equatable -> Equating
Comparable -> Comparing
Serializable -> Serializing

This started as a protocol WampSerializable and eventually became a de-protocolized contract WampSerializing which only witness now is .json, but according to WAMP should also be MessagePack (https://wamp-proto.org/_static/gen/wamp_latest.html#messagepack).

We can discuss better these naming offline to come to an agreement, for me I'm ok with any name.

Reply:
And surely I broke my own rule by calling the transport counterpart as WampTransport. And it's pretty much the same as WampSerializing but to define WebSocket, raw TCP or other possible transport contracts using de-protocolized contract. Not sure if WampTransporting would be a good name, most likely not :D